packages/components/src/placeholder/style.scss

```diff
@@ -1,15 +1,15 @@
 .components-placeholder {
+	position: relative;
```

Comment:
This is required to position and size the `resizeListener` correctly.

Reply:
That comment might good in code.